ios/FluentUI.Demo/FluentUI.Demo/Demos/ObjectiveCDemoController.m

```diff
@@ -40,18 +46,48 @@ - (void)viewDidLoad {
 
     MSFButton *testButton = [self createButtonWithTitle:@"Test" action:@selector(buttonPressed:)];
     [self.container addArrangedSubview:testButton];
+
+    [self setAddedLabels:[NSMutableSet set]];
+
+    self.appearanceController = [MSFDemoAppearanceControllerWrapper createDemoAppearanceControllerWithDelegate:nil];
```

Comment:
What's the reason for the mix-matched usage of dot notation/bracket notation throughout this file? (i.e. Why isn't this `[self setAppearanceController:[MSFDemoAppearanceControllerWrapper createDemoAppearanceControllerWithDelegate:nil]];`?)

Reply:
Yeah, unfortunately the file is already a bit inconsistent. Let me see if I can do a tiny bit of cleanup without leaving the entire file as diffs.